gulpfile.coffee

```diff
@@ -19,7 +21,7 @@ handle = (stream) ->
     do stream.end
 
 test_files = 'test/tests/*.coffee'
-coffee_files = 'assets/js/**/*'
+coffee_files = 'assets/**/*.coffee'
```

Comment:
let's do

plugin_coffee_files = 'plugins/**/*.coffee'
plugin_js_files = 'plugins/**/*.js'

since they're used in many places